### ISSUE TITLE
Unable to compile oracle_fdw

### ISSUE THREAD

**Issue by BBR1196** [laurenz/oracle_fdw#650]
Hello,
Sorry to raise a new ticket for that. I try to compile oracle_fdw on my server and I have an error.

OS : Ubuntu 22.04.3, oracle_fdw version 2.4, Postgres version 15, Oracle Client version 19.18.
Oracle client SDK installed. 

Environnement : 
ORACLE_HOME=/orabin/app/oracle/product/19.18/
LD_LIBRARY_PATH=/orabin/app/oracle/product/19.18

Error on make : 
`# make
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wimplicit-fallthrough=3 -Wcast-function-type -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -g -g -O2 -flto=auto -ffat-lto-objects -flto=auto -ffat-lto-objects -fstack-protector-strong -Wformat -Werror=format-security -fno-omit-frame-pointer -fPIC -I"/orabin/app/oracle/product/19.18//sdk/include" -I"/orabin/app/oracle/product/19.18//oci/include" -I"/orabin/app/oracle/product/19.18//rdbms/public" -I"/orabin/app/oracle/product/19.18//" -I/usr/include/oracle/21/client64 -I/usr/include/oracle/19.12/client64 -I/usr/include/oracle/19.12/client -I/usr/include/oracle/19.11/client64 -I/usr/include/oracle/19.11/client -I/usr/include/oracle/19.10/client64 -I/usr/include/oracle/19.10/client -I/usr/include/oracle/19.9/client -I/usr/include/oracle/19.9/client64 -I/usr/include/oracle/19.8/client -I/usr/include/oracle/19.8/client64 -I/usr/include/oracle/19.6/client -I/usr/include/oracle/19.6/client64 -I/usr/include/oracle/19.3/client -I/usr/include/oracle/19.3/client64 -I/usr/include/oracle/18.5/client -I/usr/include/oracle/18.5/client64 -I/usr/include/oracle/18.3/client -I/usr/include/oracle/18.3/client64 -I/usr/include/oracle/12.2/client -I/usr/include/oracle/12.2/client64 -I/usr/include/oracle/12.1/client -I/usr/include/oracle/12.1/client64 -I/usr/include/oracle/11.2/client -I/usr/include/oracle/11.2/client64 -I/usr/include/oracle/11.1/client -I/usr/include/oracle/11.1/client64 -I/usr/include/oracle/10.2.0.5/client -I/usr/include/oracle/10.2.0.5/client64 -I/usr/include/oracle/10.2.0.4/client -I/usr/include/oracle/10.2.0.4/client64 -I/usr/include/oracle/10.2.0.3/client -I/usr/include/oracle/10.2.0.3/client64 -I. -I./ -I/usr/include/postgresql/15/server -I/usr/include/postgresql/internal  -Wdate-time -D_FORTIFY_SOURCE=2 -D_GNU_SOURCE -I/usr/include/libxml2   -c -o oracle_fdw.o oracle_fdw.c
oracle_fdw.c: In function ‘oracle_fdw_validator’:
oracle_fdw.c:500:51: error: ‘Value’ undeclared (first use in this function)
  500 |                         (void)getIsolationLevel(((Value *)(def->arg))->val.str);
      |                                                   ^~~~~
oracle_fdw.c:500:51: note: each undeclared identifier is reported only once for each function it appears in
oracle_fdw.c:500:58: error: expected expression before ‘)’ token
...
...
...
`

I have other errors after that. I put all the output in this file.
[make.txt](https://github.com/laurenz/oracle_fdw/files/14333304/make.txt)

Could please have a look and see if I do something wrong ?

Thank you,

**Comment by mkgrgis** [comment]
Hello, @BBR1196! How did you get the code? If this is `git` code what is hash code of last commit?

**Comment by laurenz** [comment]
This is a duplicate of #607.

You need to use a later version of oracle_fdw with PostgreSQL v15, ideally the most recent one.

**Comment by BBR1196** [comment]
Hello,
Thanks for your quick reply. 
Indeed, the latest version works fine.

Regards
